### ISSUE TITLE
[Bug]: Herblore milestone view- stardust potion image not loading

### ISSUE THREAD

**Issue by StefanG-One** [MelvorIdle/melvoridle.github.io#1832]
### Describe the bug

In herblore skill, in the view milestones for this skill window( medal button on right side) the image for the Secret stardust potion is not loaded corectly
![image](https://user-images.githubusercontent.com/84723629/143859024-8c0fa11a-f734-4627-b413-e70c00389ff1.png)

### Reproduction Steps

1. Open the game form steam
2. Go to herblore skill
2. Click on view milestones for this skill( medal button)
3. look at secret stardust potion
Note: current skill level: 38( might not be important)

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [ ] Mobile App (iOS)
- [ ] Mobile App (Android)
- [X] Steam
- [ ] Other (Please Specify)

### Which version of the game are you experiencing this bug on?

v1.0

### Game Subversion

?1352

### Are you using any scripts or extensions?

No

**Comment by mikeaag** [comment]
I got the same missing image after returning from being offline and training herblore. It showed me what I had gained while i was away, and then showed the new potions i could make due to leveling up. `Secret Stardust Potion` was missing it's image (it was attempting to get it from `https://cdn.melvor.net/core/v018/assets/media/bank/secret_stardust_potion_i.png`).

The image displays correctly on the "Spend Mastery XP" popup (getting the image from `https://cdn.melvor.net/core/v018/assets/media/bank/potion_astrology_i.png`)

I'm using version V1.0 ?1366 in Chrome
